### PR TITLE
Resource Card component 

### PR DESCRIPTION
**Pull request by admtomas**
Component is ready for inspection.

### CODE REVIEW

File: src/components/ResourceCard/ResourceCard.styles.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  flex-direction: column;
+  align-items: center;
+
+  background: var(--white);
+  border: ${({ color }) => `0.1rem solid var(--${color})`};
+  border-radius: var(--smallest-space);
+
+  overflow: hidden;
+  position: relative;
+  width: 32.8rem;
```
Comment (r1903) [inline src/components/ResourceCard/ResourceCard.styles.js]: It is not working properly in small mobile mode, do we have to change it to max-width?

**Review by r1903 — COMMENTED** [review]
I can see a banner in the middle of the card, not sure is it only for me. Apart from that Resource card looks perfect.

File: src/components/ResourceCard/ResourceCard.styles.js (hunk 1 above)
Comment (admtomas) [reply src/components/ResourceCard/ResourceCard.styles.js]: I refactored the code. I set `max-width: 60rem;` for the container and removed some redundant code from wrapper holding Banner so now it is only for` opacity` purpose. I think now all should be fine with no wierd positioning.

File: src/components/ResourceCard/ResourceCard.styles.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Banner = styled.div`
+  opacity: 0.6;
```
Comment (r1903) [inline src/components/ResourceCard/ResourceCard.styles.js]: It is just working in mobile mode. Please add width = 100%; then it works in all viewport.

**Review by r1903 — APPROVED** [review]
If you make below changes then it is ready to go.

**Pull request merged**